common/culture/eras/00_culture_eras.txt

```diff
@@ -25,3 +25,8 @@ culture_era_late_medieval = {
 		mercenary_hire_cost_mult = 0.15
 	}
 }
+
+culture_era_early_modern = {
+	year = 800
```

Comment:
This year works as a blocker.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123692303-b7188280-d867-11eb-9c41-294c27210884.png)

</details>
So if you want gnomes and goblins there, you should lower it to 550 or something.

Reply:
![image](https://user-images.githubusercontent.com/74790101/123716954-d7d1df80-d849-11eb-8f82-a595e9a74223.png)
Would 550 work if these eras are higher than 550?

Reply:
> Would 550 work if these eras are higher than 550?

No, you need prev eras.
Also, you forgot the mercenary cost modifier.